FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift

```diff
@@ -22,6 +22,9 @@ import Foundation
     /// Used to obtain an auth credential via a mobile web flow.
     /// This method is available on iOS only.
     /// - Parameter uiDelegate: An optional UI delegate used to present the mobile web flow.
+    /// - Parameter completionHandler: Optionally; a block which is invoked asynchronously on the
+    /// main
+    /// thread when the mobile web flow is completed.
     @available(iOS 13, tvOS 13, macOS 10.15, watchOS 8, *)
     @objc(getCredentialWithUIDelegate:completion:)
```

Comment:
it's a bit unexpected but Apple generates the parameter name to be `completionHandler`, not `completion`:

```objc
getCredentialWithUIDelegate: /* type */uiDelegate completion:/* type */completion>>Handler<<
```